server.js

```diff
@@ -2,22 +2,86 @@ const express = require("express");
 const cors = require("cors");
 
 const app = express();
+app.use(express.urlencoded({ extended: true }));
 
 app.use(cors());
+app.use(express.json());
+
+app.get("/", function (request, response) {
+	response.sendFile(__dirname + "./index.html");
```

Comment:
I got an error starting up your application because of the `.` in this line. You should remove the `.`
```
Error: ENOENT: no such file or directory, stat '/Users/louisechow/workspace/node/node-challenge-chat-server./index.html'
```